examples/twitter_inlet.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import tweepy
+from databay import Inlet, Link, Record
+from databay.outlets import PrintOutlet
+from databay.planners import ApsPlanner, SchedulePlanner
+from tweepy.models import Status
+
+
+class TwitterTimelineInlet(Inlet):
+
+    def __init__(self, api: tweepy.API, most_recent_id=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.api = api
+
+        # this will ensure we only every pull tweets that haven't been handled
+        self.most_recent_id = most_recent_id
+
+    def pull(self, update):
+        if self.most_recent_id is not None:
+            public_tweets = self.api.home_timeline(
+                since_id=self.most_recent_id)
+        else:
+            public_tweets = self.api.home_timeline()
+
+        if len(public_tweets) > 0:
+            # 0th tweet is most recent
+            self.most_recent_id = public_tweets[0].id
+
+        tweets = []
+        for tweet in public_tweets:
+            tweets.append({"user": tweet.user.screen_name, "text": tweet.text})
+        return tweets
+
+
+class TwitterUserInlet(Inlet):
+    def __init__(self, api: tweepy.API, user: str, most_recent_id=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.api = api
+        self.user = user
+
+        # this will ensure we only every pull tweets that haven't been handled
+        self.most_recent_id = most_recent_id
+
+    def pull(self, update):
+        if self.most_recent_id is not None:
+            public_tweets = self.api.user_timeline(
+                self.user, since_id=self.most_recent_id)
+        else:
+            public_tweets = self.api.user_timeline(
+                self.user)
+        if len(public_tweets) > 0:
+            # 0th tweet is most recent
+            self.most_recent_id = public_tweets[0].id
+        tweets = []
+        for tweet in public_tweets:
+            tweets.append({"user": tweet.user.screen_name, "text": tweet.text})
+
+        return tweets
+
+
+# gets twitter api secrets and keys from environment vars
+consumer_key = os.getenv("twitter_key")
```

Comment:
`os.getenv` ❤️ 

I always used `os.environ.get`, didn't know about this alias!